journal/modernized/38-chapter-35.adoc

```diff
@@ -845,6 +846,7 @@ was satisfactory; and afterwards the Meeting for Sufferings.
 
 19th. (First Day.)--The people collected for the forenoon meeting in great numbers.
 The neighbourhood of the meeting house exhibited such a display of equipages,
+//equipages is archaic, but I'm not sure what definiton Wheeler was meaning here. Possibly horse drawn carriages? 
```

Comment:
yeah, i'm not sure what to do. i think let's just leave it for now. i'm a bit afraid to guess and be wrong.